### PR TITLE
Added some missing parameters to camera.

### PR DESCRIPTION
**Pull request by GuillaumeSchmid**
* auto_wb
* wb_kr
* wb_kg
* wb_kb

### CODE REVIEW

**Comment by superbort** [comment]
This looks good to me and I will merge it as soon as I am able to run the tests with a camera connected. This may however still take a few days, since I don't have physical access to the cameras at the moment. Thank you for your contributions!

File: src/camera.rs [hunk 1]
```diff
@@ -586,6 +586,23 @@ impl Camera {
         /// Read the sensor clock frequency in Hz
         sensor_clock_freq_hz: f32;
 
+        /// Data move policy
+        mut buffer_policy: i32;
+
+        /// Auto white balance mode.
+        mut auto_wb: i32;
```
Comment (superbort) [inline src/camera.rs]:
```suggestion
        mut auto_wb: XI_SWITCH::Type;
```
The parameter is a simple on/off switch, so it should have the appropriate type for that.

File: src/camera.rs [hunk 2]
```diff
@@ -586,6 +586,23 @@ impl Camera {
...
+        mut wb_kr: f32;
+
+        /// White balance Green coefficient.
+        mut wb_kg: f32;
+
+        /// White balance Blue coefficient.
+        mut wb_kb: f32;
+
+        /// Recent Frame mode.
+        mut recent_frame: u32;
```
Comment (superbort) [inline src/camera.rs]:
```suggestion
        mut recent_frame: XI_SWITCH::Type;
```
Same as above.

**Review by superbort — CHANGES_REQUESTED** [review]
Can you adjust the types for the simple on/off switches? They are represented as i32 on the lower level, but they will only ever be 0 or 1, so they can be XI_SWITCH in the high level bindings.
There should also be a more appropriate type for the buffer policy, but it is missing in the xiapi_sys bindings, so i32 is okay for now.

**Comment by GuillaumeSchmid** [comment]
I made the changes you requested.

**Pull request merged**